datapackage/group.py

```diff
@@ -52,3 +53,24 @@ def read(self, limit=None, **options):
             if count == limit:
                 break
         return rows
+
+    def check_relations(self):
```

Comment:
Could you please add a test for this method - https://coveralls.io/builds/26209815/source?filename=datapackage%2Fgroup.py#L62?

Reply:
Yep almost done.